sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsAsyncClient.java

```diff
@@ -626,4 +614,11 @@ private Map<String, ParticipantProperties> convertRoomIdentifiersToMapForRemove(
         return participantMap;
     }
 
+    private static String getUpdateRequest(Map<String, ParticipantProperties> participantMap) throws IOException {
+        Writer json = new StringWriter();
+        try (JsonWriter jsonWriter = JsonProviders.createWriter(json)) {
+            new UpdateParticipantsRequest().setParticipants(participantMap).toJson(jsonWriter);
```

Comment:
Maybe this would be more readable?
```suggestion
            UpdateParticipantsRequest updateRequest = new UpdateParticipantsRequest().setParticipants(participantMap);
            jsonWriter.writeJson(updateRequest).flush();
```

(the essential part is to call `flush`, the rest is cosmetic)